data/sql/updates/pending_db_world/rev_1585394899832297800.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1585394899832297800');
+
+UPDATE `creature` SET `position_x` = 1488.602661, `position_y` = 6376.107910, `position_z` = 35.148029, `spawndist` = 0, `MovementType` = 0 WHERE `guid` = 71130; 
```

Comment:
Update this since spawndist got renamed to wander_distance in https://github.com/azerothcore/azerothcore-wotlk/commit/8a1eab2c2330537a36e1f1fdae56ea9ad40b5e43
```suggestion
UPDATE `creature` SET `position_x` = 1488.602661, `position_y` = 6376.107910, `position_z` = 35.148029, `wander_distance` = 0, `MovementType` = 0 WHERE `guid` = 71130; 
```